build.gradle

```diff
@@ -81,13 +82,13 @@ task generateTestScript {
     description 'Generates a test.sh to runs every guide test at build/code'
     ext {
         excludeDirs = ['images']
-        stopIfFailure = false
+        stopIfFailure = true
```

Comment:
Why have you changed this? I think it makes more sense to run always all the tests of all the guides.

Reply:
I changed it accidentally. I was testing locally and I forgot to change it back. I have reverted the change.